dd-java-agent/instrumentation/lettuce-5/src/main/java8/datadog/trace/instrumentation/lettuce/LettuceClientDecorator.java

```diff
@@ -44,6 +44,10 @@ protected String dbUser(final RedisURI connection) {
 
   @Override
   protected String dbInstance(final RedisURI connection) {
+    return null;
```

Comment:
why wouldn't this be `connection.getDatabase()`?

Reply:
It returns a number, an integer. Is this (a number) what is preferred?

Reply:
Yes, the initial/default connection is for db 0 and people are able to configure more as their application, deployment, and upgrade patterns require.